features/steps/fhir.py

```diff
@@ -9,9 +9,25 @@
 ERROR_FIELD_UNEXPECTED_VALUE = """
 Resource field '{field_name}' does not match expected '{expected}', got '{actual}'.
 """
+ERROR_NO_NEXT_LINK = 'Link with relation "next" not found.'
 ERROR_UNRESOLVED_REFERENCE = "Reference '{reference}' failed to resolve."
 
 
+@when('I follow the "next" link')
+def step_impl(context):
+    resource = context.response.json()
+
+    links = resource.get('link', [])
+    urls = [link['url'] for link in links
+            if link['relation'] == 'next']
+
+    assert len(urls) == 1, \
```

Comment:
I don't think we want to go quite as far as requiring that there be a next link. But we do want to test the next link if it is present. The only failing condition should be a next link that is present but broken.